net/Socket.cpp

```diff
@@ -1303,13 +1306,18 @@ bool StreamSocket::parseHeader(const char *clientName,
 {
     assert(map._headerSize == 0 && map._messageSize == 0);
 
+    constexpr std::chrono::duration<float, std::milli> delayMax =
+        std::chrono::duration_cast<std::chrono::milliseconds>(SocketPoll::DefaultPollTimeoutMicroS);
+
+    std::chrono::steady_clock::time_point now = std::chrono::steady_clock::now();
+    std::chrono::duration<float, std::milli> delayMs = now - _lastSeenHTTPHeader;
+
     // Find the end of the header, if any.
     static const std::string marker("\r\n\r\n");
-    auto itBody = std::search(_inBuffer.begin(), _inBuffer.end(),
-                              marker.begin(), marker.end());
+    auto itBody = std::search(_inBuffer.begin(), _inBuffer.end(), marker.begin(), marker.end());
     if (itBody == _inBuffer.end())
     {
-        LOG_TRC(clientName << " doesn't have enough data for the header yet.");
+        LOG_TRC("parseHeader: " << clientName << " doesn't have enough data for the header yet. delay " << delayMs.count() << "ms");
```

Comment:
If `delayMs` is in a standard unit (i.e. `chrono::millseconds`, `chrono::microseconds`, etc.), you don't need the `.count()` nor the `<< "ms"`, instead, just stream `delayMs` and you get the units too! (Yet another reason to avoid floating-point duration).